File: 0_patterns/pattern-2.py
# Given an integer N, print the following pattern.
# 1
# 12
# 123
# 1234
# 12345

def printPattern(N):
    for i in range(N):
        pattern = ""
        for j in range(i+1):
            pattern += str(j+1)
        print(pattern)
    
printPattern(5)

File: 0_patterns/pattern-2.js
// Given an integer N, print the following pattern.
// 1
// 12
// 123
// 1234
// 12345

const printPattern = (N) => {
    for (let row = 1; row <= N; row++) {
        let patternStr = "";
        for (let col = 1; col <= row; col++) {
            patternStr += col
        }
        console.log(patternStr)
    }
}

printPattern(5)


